R/prepare_sector_split.R

```diff
@@ -30,47 +30,37 @@
 # The intended use case is to provide a rule by which to allocate loans to
 # multiple sectors, for loan books where such loan allocation is ambiguous.
 
-prepare_sector_split <- function() {
-  # load config----
-  config_dir <- config::get("directories")
-  config_files <- config::get("file_names")
-  config_project_parameters <- config::get("project_parameters")
-  config_prepare_sector_split <- config::get("sector_split")
-
-  dir_matched <- config_dir$dir_matched
-
-  path_sector_split <- file.path(
-    config_prepare_sector_split$dir_split_company_id,
-    config_prepare_sector_split$filename_split_company_id
-  )
+prepare_sector_split <- function(config) {
+  config <- load_config(config)
 
-  path_advanced_company_indicators <- file.path(
-    config_prepare_sector_split$dir_advanced_company_indicators,
-    config_prepare_sector_split$filename_advanced_company_indicators
-  )
+  dir_matched <- get_matched_dir(config)
+  abcd_dir <- get_abcd_dir(config)
```

Comment:
nit: naming convention seems to be inversed. should be `dir_abcd`

Reply:
let's discuss naming conventions on Wednesday and then implement everywher

Reply:
- #54

Reply:
I'd like to join the naming convention call too!